client/src/main/java/io/spine/quickstart/client/ClientApp.java

```diff
@@ -0,0 +1,156 @@
+import io.grpc.ManagedChannelBuilder;
+import io.spine.Identifier;
+import io.spine.client.ActorRequestFactory;
+import io.spine.client.Query;
+import io.spine.client.QueryResponse;
+import io.spine.client.grpc.CommandServiceGrpc;
+import io.spine.client.grpc.QueryServiceGrpc;
+import io.spine.core.Ack;
+import io.spine.core.Command;
+import io.spine.core.UserId;
+import io.spine.quickstart.Task;
+import io.spine.quickstart.c.CreateTask;
+import io.spine.serverapp.TaskId;
+import io.spine.string.Stringifiers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A template of a client for Spine-powered server.
+ *
+ * <p>Illustrates a simple flow:
+ *
+ * <ul>
+ *      <li>establishes a connection to the gRPC server;
+ *      <li>sends a command to create a task through {@code CommandService};
+ *      <li>verifies that the task is created by asking for all tasks via {@code QueryService}.
+ * </ul>
+ *
+ * @author Alex Tymchenko
+ */
+public class ClientApp {
+
+    /**
+     * A host of the gRPC server to connect.
+     */
+    private static final String HOST = "127.0.0.1";
+
+    /**
+     * A port at which gRPC server listens for connections.
+     */
+    private static final int PORT = 8484;
+
+    /**
+     * Prevent this class from instantiation.
+     */
+    private ClientApp() {}
+
+    /**
+     * Connects to the gRPC server and interacts with it via exposed gRPC services.
+     *
+     * <p>Uses the hard-coded {@linkplain #HOST host} and {@linkplain #PORT port} for simplicity.
+     */
+    public static void main(String[] args) {
+
+        // Connect to the server and init the client-side stubs for gRPC services.
+        log().info("Connecting to the server at {}:{}", HOST, PORT);
+
+        final ManagedChannel channel = ManagedChannelBuilder.forAddress(HOST, PORT)
+                                                            .usePlaintext(true)
+                                                            .build();
+        final CommandServiceGrpc.CommandServiceBlockingStub clientCommandService =
+                CommandServiceGrpc.newBlockingStub(channel);
+
+        final QueryServiceGrpc.QueryServiceBlockingStub queryClientService =
+                QueryServiceGrpc.newBlockingStub(channel);
+
+        // Create and post a command.
+        final ActorRequestFactory requestFactory = ActorRequestFactory.newBuilder()
+                                                                      .setActor(whoIsCalling())
+                                                                      .build();
+        final CreateTask createTask = newCreateTaskMsg("Wash my car");
+        final Command cmd = requestFactory.command()
+                                          .create(createTask);
+
+        final Ack acked = clientCommandService.post(cmd);
+        log().info("A command has been posted: " + Stringifiers.toString(createTask));
+        log().info("(command acknowledgement: {})", Stringifiers.toString(acked));
+
```

Comment:
Don't we want a delay or something in between? Or, a comment saying that in real life it's going to be an event or another update from the server?

Reply:
A comment has been added.